src/pages/Staking/index.tsx

```diff
@@ -600,10 +615,7 @@ const Staking = (): JSX.Element => {
 
   const renderUnlockDateLabel = () => {
     const numericLockTime = parseInt(lockTime);
-    if (
-      numericLockTime < STAKE_LOCK_TIME.MIN ||
-      numericLockTime > STAKE_LOCK_TIME.MAX
-    ) {
+    if (errors[LOCK_TIME]) {
```

Comment:
According to the single source of truth principle, the relevant validation is handled in a single place and this function just references it without duplicating the validation logic inside its body.
Besides that, the validation logic had a mistake. Probably because it was duplicated, it was not updated properly. That's why I followed the single source of truth this time.